packages/devtools-reps/src/reps/tests/element-node.js

```diff
@@ -172,6 +172,35 @@ describe("ElementNode - Node with leading and trailing spaces class name", () =>
   });
 });
 
+describe("ElementNode - Node with spaces in the class name", () => {
+  const stub = stubs.get("NodeWithSpacesInClassName");
+
+  it("selects ElementNode Rep", () => {
+    expect(getRep(stub)).toBe(ElementNode.rep);
+  });
+
+  it("renders with expected text content", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub
+    }));
+
+    // TODO: This should be <body class=\"a  b   c\"> but enzyme -> shallow -> text()
```

Comment:
Yeah, I also saw some weirdness with `wrapper.text()` at some point.
Maybe we can be a bit paranoid and add a snapshot on the rendered component so we don't miss any regression ? 

```js
expect(renderedComponent.debug()).toMatchSnapshot();
```